internal/auth0/user/resource.go

```diff
@@ -153,6 +153,9 @@ func NewResource() *schema.Resource {
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				Description: "Set of IDs of roles assigned to the user.",
+				Deprecated: "This attribute is deprecated and it will be removed in a future version. " +
```

Comment:
Might be worth making the distinction of it becoming read-only instead of removed.